packages/p2p/src/stores/advertiser-page-store.js

```diff
@@ -15,6 +15,7 @@ export default class AdvertiserPageStore extends BaseStore {
     @observable api_error_message = '';
     @observable form_error_message = '';
     @observable has_more_adverts_to_load = false;
+    @observable is_block_user_modal_open = true;
```

Comment:
Should this be initialised with `true`?

Reply:
Ah, I need to refactor this, this should be false. I forgot to refactor this back to false when I was testing. Will change it!